notification-server/src/models/device.rs

```diff
@@ -0,0 +1,42 @@
+use enum_iterator::Sequence;
+use serde::{Deserialize, Serialize};
+use std::collections::HashSet;
+
+#[derive(Debug, Deserialize)]
+pub struct DeviceSubscription {
+    pub preferences: Vec<Preference>,
+    pub accounts:    Vec<String>,
```

Comment:
We probably discussed this before, but why do we represent accounts as `String` and not `AccountAddress`? As I see it, this seems to be used for request parameters but why not get the conversion from base58string -> `AccountAddress` without manually parsing (axum automatically deserializes, but I assume you know this)?

Reply:
I cannot remember. I can look into this in another PR. 

This is simply a move of a file. I would prefer not to touch the API in this PR. :)

Reply:
Okay, maybe add a comment to change it then? 😄